/usr/bin/python3: can't open file '/home/soda/vla/ROS_action/build/vla_node/setup.py': [Errno 2] No such file or directory
